mino/option_test.go

```diff
@@ -33,20 +33,44 @@ func TestFilter_RotateFilter(t *testing.T) {
 
 func TestFilter_IndexFilter(t *testing.T) {
 	filters := &Filter{Indices: []int{}}
-
 	IndexFilter(1)(filters)
-	require.Equal(t, filters.Indices, []int{1})
+	require.Equal(t, []int{1}, filters.Indices)
+
+	IndexFilter(2)(filters)
+	require.Equal(t, []int{1, 2}, filters.Indices)
 
 	IndexFilter(2)(filters)
-	require.Equal(t, filters.Indices, []int{1, 2})
+	require.Equal(t, []int{1, 2}, filters.Indices)
 
 	IndexFilter(0)(filters)
-	require.Equal(t, filters.Indices, []int{0, 1, 2})
+	require.Equal(t, []int{0, 1, 2}, filters.Indices)
 
 	IndexFilter(0)(filters)
 	IndexFilter(1)(filters)
 	IndexFilter(2)(filters)
-	require.Equal(t, filters.Indices, []int{0, 1, 2})
+	require.Equal(t, []int{0, 1, 2}, filters.Indices)
+}
+
+func TestFilter_RejectFilter(t *testing.T) {
+	filters := &Filter{Indices: []int{1, 2, 3, 4}}
+
+	testCases := []struct {
+		filterVal int
+		expected  []int
+	}{
+		{0, []int{1, 2, 3, 4}},
+		{5, []int{1, 2, 3, 4}},
+		{2, []int{1, 3, 4}},
+		{1, []int{3, 4}},
+		{4, []int{3}},
+		{4, []int{3}},
```

Comment:
duplicate line? also isn't that testing the same case as ln. 62 (index out of left bound)?

Reply:
It's not a duplicate line, but it's testing the same case as in line 62, yes. So I'll remove it.